libs/vm-modules/src/math/tensor/tensor_estimator.cpp

```diff
@@ -59,9 +59,17 @@ ChargeAmount TensorEstimator::Copy()
   SizeType padded_size = fetch::math::Tensor<DataType>::PaddedSizeFromShape(tensor_.shape());
   SizeType size        = fetch::math::Tensor<DataType>::SizeFromShape(tensor_.shape());
 
-  return static_cast<ChargeAmount>(DEFAULT_PADDED_SIZE_COEF * padded_size +
-                                   DEFAULT_SIZE_COEF * size + DEFAULT_CONST_COEF) *
-         COMPUTE_CHARGE_COST;
+  auto ret = static_cast<ChargeAmount>(DEFAULT_PADDED_SIZE_COEF * padded_size +
+                                       DEFAULT_SIZE_COEF * size + DEFAULT_CONST_COEF) *
+             COMPUTE_CHARGE_COST;
+
+  // Ensure that estimate will never be 0
```

Comment:
since you repeat this code a lot why not just wrap it up in a private function called "PreventZeroCharge"

Reply:
I named that ToChargeAmount...